docs/References.bib

```diff
@@ -1,112 +1,314 @@
+}
+
+@article{Owen2017yaj,
+  author         = "Owen, Robert and Fox, Alex S. and Freiberg, John A. and
+                    Jacques, Terrence Pierre",
+  title          = "Black Hole Spin Axis in Numerical Relativity",
+  year           = "2017",
+  eprint         = "1708.07325",
+  archivePrefix  = "arXiv",
+  primaryClass   = "gr-qc",
+  SLACcitation   = "%%CITATION = ARXIV:1708.07325;%%"
+}
+
+@article{Porth2016rfi,
+  author         = "Porth, Oliver and Olivares, Hector and Mizuno, Yosuke and
+                    Younsi, Ziri and Rezzolla, Luciano and Moscibrodzka,
+                    Monika and Falcke, Heino and Kramer, Michael",
+  title          = "The Black Hole Accretion Code",
+  url            = "https://doi.org/10.1186/s40668-017-0020-2",
+  year           = "2016",
+  eprint         = "1611.09720",
+  archivePrefix  = "arXiv",
+  primaryClass   = "gr-qc",
+  SLACcitation   = "%%CITATION = ARXIV:1611.09720;%%"
 }
 
 @article{Stamm2010,
-  author  = {Stamm, Benjamin and Wihler, Thomas P.},
-  title   = {hp-Optimal discontinuous {Galerkin} methods for linear elliptic problems},
-  year    = {2010},
-  url     = {https://doi.org/10.1090/S0025-5718-10-02335-5},
-  doi     = {10.1090/S0025-5718-10-02335-5},
-  journal = {Mathematics of Computation},
-  pages   = {2117--2133},
-  volume  = {79},
-}
-
-@article{Lindblom:2005qh,
-      author         = "Lindblom, Lee and Scheel, Mark A. and Kidder, Lawrence E.
-                        and Owen, Robert and Rinne, Oliver",
-      title          = "{A New generalized harmonic evolution system}",
-      journal        = "Class. Quant. Grav.",
-      volume         = "23",
-      year           = "2006",
-      pages          = "S447-S462",
-      doi            = "10.1088/0264-9381/23/16/S09",
-      eprint         = "gr-qc/0512093",
-      archivePrefix  = "arXiv",
-      primaryClass   = "gr-qc",
-      url            = {https://doi.org/10.1088/0264-9381/23/16/S09},
-      SLACcitation   = "%%CITATION = GR-QC/0512093;%%"
+  author  = "Stamm, Benjamin and Wihler, Thomas P.",
+  title   = "{hp-Optimal} discontinuous {Galerkin} methods for linear elliptic
+             problems",
+  year    = "2010",
+  url     = "https://doi.org/10.1090/S0025-5718-10-02335-5",
+  doi     = "10.1090/S0025-5718-10-02335-5",
+  journal = "Mathematics of Computation",
+  pages   = "2117--2133",
+  volume  = "79",
+}
+
+@article{Teukolsky2015ega,
+  author         = "Teukolsky, Saul A.",
+  title          = "Formulation of discontinuous {Galerkin} methods for
+                    relativistic astrophysics",
+  journal        = "J. Comput. Phys.",
+  volume         = "312",
+  year           = "2016",
+  pages          = "333-356",
+  doi            = "10.1016/j.jcp.2016.02.031",
+  url            = "https://doi.org/10.1016/j.jcp.2016.02.031",
+  eprint         = "1510.01190",
+  archivePrefix  = "arXiv",
+  primaryClass   = "gr-qc",
+  SLACcitation   = "%%CITATION = ARXIV:1510.01190;%%"
```

Comment:
Please add the comment about the formatting at the end too. I can see people just adding things at the end without reading the comment at the beginning (myself included in that list :) )